react-components/src/utilities/constants.ts

```diff
@@ -5,3 +5,4 @@
 export const DEFAULT_QUERY_STALE_TIME = 1000 * 60 * 10; // 10 minutes
 export const METERS_TO_FEET = 3.28084;
 export const FEET_TO_INCHES = 12;
+export const EMPTY_ARRAY = [];
```

Comment:
```suggestion
export const EMPTY_ARRAY = [] as const;
```
Just to make sure no-one can append anything to it